import-data.sh

```diff
@@ -19,12 +19,9 @@ WORKING_DIR=$TMP_DIR/work
 DATA_DIR=$WORKING_DIR/data
 TEAM_IMAGES_DIR=$DATA_DIR/team_images
 
-# get data version from agora repo
-wget https://raw.githubusercontent.com/Sage-Bionetworks/Agora/$TRAVIS_BRANCH/package.json -O $WORKING_DIR/package-$TRAVIS_BRANCH.json
-
 # Version key/value should be on his own line
-DATA_VERSION=$(cat $WORKING_DIR/package-$TRAVIS_BRANCH.json | grep data-version | head -1 | awk -F: '{ print $2 }' | sed 's/[",]//g' | tr -d '[[:space:]]')
-echo "package-$TRAVIS_BRANCH.json DATA_VERSION = $DATA_VERSION"
+DATA_VERSION=$(cat data-manifest.json | grep data-version | head -1 | awk -F: '{ print $2 }' | sed 's/[",]//g' | tr -d '[[:space:]]')
+echo "$TRAVIS_BRANCH branch, DATA_VERSION = $DATA_VERSION"
 
 # get data from synapse
 synapse -u $SYNAPSE_USERNAME -p $SYNAPSE_PASSWORD cat --version $DATA_VERSION syn18387112 | tail -n +2 | while IFS=, read -r id version; do
```

Comment:
I thought we were going to parameterize the synapse entity id as well?  Maybe you want to add that to the data-manifest.json?

Wondering if the synapse entity for team images should be added to data-manifest as well?

Reply:
Sure, makes sense I can add them as new properties in the `data-manifest.json`. I'll also change the id to the live data one.